app/views/main.scala.html

```diff
@@ -15,13 +17,32 @@
         <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/favicon.png")">
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
         <script src="@routes.Assets.versioned("javascripts/main.js")" type="text/javascript"></script>
+        <script type="text/javascript">
+            window.guardian = {
+                'urls': {
+                    @if(stage == PROD) {
```

Comment:
I may be missing something, but what about when `stage === CODE`?

Reply:
there's an `else` block below

Reply:
But the else block contains dev URLs, not CODE ones (doesn't it?!)

Reply:
oh I see what you mean. This confused me but is consistent with the original logic! Tbh I struggled to find where these urls are even used!

Reply:
Ahh okay fair enough! 👍